test/unit/flux/t0001-flux.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from ats.tests import AtsTest
+
+class TestFluxScheduled(unittest.TestCase):
+    @classmethod
+    def setUp():
+        self.test = AtsTest()
+
+    def test_command_list(self, test: AtsTest):
+        return 0
+
+    def test_create_flux_instance(self):
+        return 0
```

Comment:
Tests don't need to return anything. If a test doesn't fail an assertion or raise an exception it is considered to have succeeded.